etc/parameters/default.json

```diff
@@ -809,20 +809,23 @@
       "position": [-0.23, 0.05],
       "orientation": 0.0,
       "reached_thresholds": [0.06, 0.03, 0.1],
+      "precision_kick_reached_thresholds": [0.01, 0.015, 0.03],
       "shot_distance": 4.0,
       "enabled": true
     },
     "turn": {
       "position": [-0.176, 0.09],
       "orientation": -1.0,
       "reached_thresholds": [0.04, 0.04, 0.1],
+      "precision_kick_reached_thresholds": [0.0, 0.0, 0.0],
```

Comment:
This seems like a hack, same below

Reply:
The numbers here are never read, since these kicks are not generated in the applicable cases. I like it this way, since it enables 
```
let thresholds = if precision_kick {
        kick_info.precision_kick_reached_thresholds
    } else {
        kick_info.reached_thresholds
    };
```

Reply:
That's exactly my point. The values are never read. In my opinion they should not be in the parameters.
You could make the field an `Option<_>` if you want to keep it in the struct

Reply:
I think we should act more pragmatically more often ;) Let's discuss